metricbeat/helper/prometheus/hash.go

```diff
@@ -0,0 +1,71 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package prometheus
+
+import (
+	"bytes"
+	"sort"
+	"strconv"
+	"sync"
+
+	"github.com/cespare/xxhash"
```

Comment:
We already have an xxhash implementation in the vendor/ dir. Can that one be used instead of adding another? If not what's the reason (maybe we need to change the other users to this version)?

Reply:
@andrewkroh I am changing the auditbeat modules to use this one as there is documentation on the cespare based on being better performant. we did some internal benchmarks and we use the same for our internal needs as well

Reply:
SGTM